src/components/InputSwitch.vue

```diff
@@ -16,7 +17,11 @@ const emit = defineEmits(['update:modelValue']);
   <div class="flex items-center space-x-2 pt-1 pr-2">
     <Switch
       :model-value="modelValue"
-      :class="modelValue ? 'bg-green' : 'bg-skin-border'"
+      :class="[
+        modelValue ? 'bg-green' : 'bg-skin-border',
+        { 'cursor-not-allowed': isDisabled }
+      ]"
+      :disabled="isDisabled"
```

Comment:
Oh we already have it 😅 Just getting an override

Reply:
Yep thanks fixed that